docs/userguide/aggregated-api-endpoint.md

```diff
@@ -80,7 +80,7 @@ Or append `/apis/cluster.karmada.io/v1alpha1/clusters/{clustername}/proxy ` to t
 kubectl --kubeconfig karmada-apiserver.config get node
 ```
 
-> Note: For a member cluster that joins karmada in pull mode, we can [deploy apiserver-network-proxy (ANP)](../working-with-anp.md) to access it.
+> Note: For a member cluster that joins karmada in pull mode, if it uses a one-way network, we can [deploy apiserver-network-proxy (ANP)](../working-with-anp.md) to access it.
```

Comment:
`if it uses a one-way network,`

I think we should give more clear comments here. 
cc @Tingtal for help
Here want to say is `maybe the cluster is behind a NAT and Karmada control plane can't access it directly.

Reply:
I don't think there is a term to specifically describe such a situation... 
For me, I would say,
```suggestion
> Note: A member cluster that joins karmada in pull mode allows only cluster-to-karmada access. In this case, we can [deploy apiserver-network-proxy (ANP)](../working-with-anp.md) to access the cluster.
```

But here's another doubt. You said "**maybe** the cluster is behind a NAT". This means that there are other reasons for such one-way communication and **not all** member clusters in pull mode deny the access requests from the Karmada control plane, right?
If that's the case, I would say,

```suggestion
> Note: For a member cluster that joins karmada in pull mode and allows only cluster-to-karmada access, we can [deploy apiserver-network-proxy (ANP)](../working-with-anp.md) to access it.
```

Pls check my edits. Cheers!

Reply:
Thanks!

Reply:
>  You said "maybe the cluster is behind a NAT". This means that there are other reasons for such one-way communication and not all member clusters in pull mode deny the access requests from the Karmada control plane, right?

Yes, thanks!

Reply:
>  Note: For a member cluster that joins karmada in pull mode and allows only cluster-to-karmada access, we can [deploy apiserver-network-proxy (ANP)](../working-with-anp.md) to access it.

+1